astropy/units/quantity_helper/scipy_special.py

```diff
@@ -63,10 +63,6 @@ def get_scipy_special_helpers():
     import scipy.special as sps
     SCIPY_HELPERS = {}
     for name in dimensionless_to_dimensionless_sps_ufuncs:
-        # TODO: Revert https://github.com/astropy/astropy/pull/7219 when
-        #       astropy requires scipy>=0.18, and loggamma is guaranteed
```

Comment:
Pretty sure this comment is already outdated, but maybe @mhvk can confirm.

Reply:
Yes, confirmed!